nbgitpuller/hookspecs.py

```diff
@@ -0,0 +1,22 @@
+import pluggy
+
+hookspec = pluggy.HookspecMarker("nbgitpuller")
+hookimpl = pluggy.HookimplMarker("nbgitpuller")
```

Comment:
I don't understand this, an inline comment could be very helpful regarding these.

Reply:
good deal added comments